Buildings/Applications/DHC/EnergyTransferStations/CoolingIndirect.mo

```diff
@@ -0,0 +1,287 @@
+within Buildings.Applications.DHC.EnergyTransferStations;
+model CoolingIndirect
+  "Indirect cooling energy transfer station for district energy systems"
+  extends Buildings.Fluid.Interfaces.PartialFourPort(
+    redeclare package Medium1 = Medium,
+    redeclare package Medium2 = Medium);
+
+  package Medium = Buildings.Media.Water;
```

Comment:
The `Medium` package does not need to be declared here. It has (will) be declared when instantiates it, like you did in  model `Buildings.Applications.DHC.EnergyTransferStations.Examples.CoolingIndirectOpenLoops`

Reply:
Revised as suggested. I made some additional modifications with the `cp` calculation as well, as errors were introduced with this `Medium` change. The calculation of the default cp (now `cp_default`) is a `protected` `final` parameter in `CoolingIndirect.mo`. This can be seen in the next commit.